simple_network_sim/network_of_populations.py

```diff
@@ -194,6 +285,24 @@ def getWeight(graph, orig, dest, multiplier):
 # Reminder: I expect the weighted edges to be the number of *expected infectious* contacts (if the giver is infectious)
 #  We may need to multiply movement numbers by a probability of infection to achieve this.   
 def getExternalInfections(graph, dictOfStates, currentTime, movementMultiplier):
+    """Calculate the number of new infections in each region, distributed across each age group
+    relative to the size [of the number of susceptibles] of each age group.
```

Comment:
I think there's two much information in the same phrase. Perhaps: *Calculate the number of new infections in each region. The infections are distributed proportionally to the number of susceptibles in the destination node and infected in the origin node. The infections are distributed to each age group according to the number of suscepitible people in them.*